pkg/operatorstatus/operatorstatus.go

```diff
@@ -103,12 +103,6 @@ func (o *ClusterOperatorHandler) setOperatorStatus(condtype configv1.ClusterStat
 			state = &configv1.ClusterOperator{}
 			state.Name = ClusterOperatorName
 
-			state.Status.RelatedObjects = []configv1.ObjectReference{
-				{Group: v1.GroupName, Resource: "configs", Name: "cluster"},
-				{Resource: "namespaces", Name: "openshift-cluster-samples-operator"},
-				{Resource: "namespaces", Name: "openshift"},
-			}
-
 			state.Status.Conditions = []configv1.ClusterOperatorStatusCondition{
```

Comment:
@gabemontero how are the conditions set if `Status` is not set on `Create`?

Reply:
@adambkaplan it is this existing call below:

```
		modified := o.updateOperatorCondition(state, &configv1.ClusterOperatorStatusCondition{
			Type:               condtype,
			Type:               condtype,
			Status:             status,
			Status:             status,
			Message:            msg,
			Message:            msg,
			LastTransitionTime: metaapi.Now(),
			LastTransitionTime: metaapi.Now(),
		})
```

Followed by this existing call:

```
return o.ClusterOperatorWrapper.UpdateStatus(state)
```